### PR TITLE
Reposition Token Overview on Dashboard

### PR DESCRIPTION
**Pull request by csillag**
Move Token Overview above Learning Materials on mobile dashboard page.

Task is here: https://app.clickup.com/t/861n1n25y

### CODE REVIEW

**Review by buberdds — APPROVED** [review]

File: src/app/pages/ParatimeDashboardPage/index.tsx [hunk 1]
```diff
@@ -21,14 +21,19 @@ export const ParatimeDashboardPage: FC = () => {
       <Divider variant="layout" sx={{ mt: isMobile ? 4 : 0 }} />
       <LatestTransactions />
       <Grid container spacing={4}>
-        <Grid item xs={12} md={6} sx={{ display: 'flex', order: isMobile ? 1 : 0 }}>
+        <Grid item xs={12} md={6} sx={{ display: 'flex', order: isMobile ? 2 : 0 }}>
```
Comment (buberdds) [inline src/app/pages/ParatimeDashboardPage/index.tsx:24]: This change is not needed, right?

Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/index.tsx:24]: I'm not sure. I assumed that now (on mobile) we want it at the 3rd spot, after Blocks and Tokens. Isn't that the case?

File: src/app/pages/ParatimeDashboardPage/index.tsx [hunk 2]
```diff
@@ -21,14 +21,19 @@ export const ParatimeDashboardPage: FC = () => {
...
           <LearningMaterials />
         </Grid>
         <Grid item xs={12} md={6}>
           <LatestBlocks />
         </Grid>
+        {isMobile && (
+          <Grid item xs={12} md={6}>
```
Comment (buberdds) [inline src/app/pages/ParatimeDashboardPage/index.tsx:31]: we don't need md prop here

Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/index.tsx]: Indeed. Removed.

File: src/app/pages/ParatimeDashboardPage/index.tsx (hunk 1 above)
Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/index.tsx]: Right, undone.

**Pull request merged**